### ISSUE TITLE
Allow paste multiple watermarks

### ISSUE THREAD

**Issue by Koc** [liip/LiipImagineBundle#1007]
| Q | A
| --- | ---
| Bug Report? | no
| Feature Request? | yes
| BC Break Report? | no
| RFC? | no
| Imagine Bundle Version | latest

For now it is impossible insert multiple watermarks because filter name is key of the array.

**Comment by peter-gribanov** [comment]
You can solve this problem by creating a additional filter

```yml
services:
    liip_imagine.filter.loader.watermark2:
        class: '%liip_imagine.filter.loader.watermark.class%'
        arguments: [ '@liip_imagine', '%kernel.root_dir%' ]
        tags:
            - { name: liip_imagine.filter.loader, loader: 'watermark2' }
```

and apply it

```yml
liip_imagine:
    filter_sets:
        my_watermark_filter:
            filters:
                watermark:
                    image: 'assets/images/watermark1.png'
                    position: 'topleft'
                watermark2:
                    image: 'assets/images/watermark2.png'
                    position: 'bottomright'
```
But this does not eliminate need to resolve the problem.

**Comment by dbu** [comment]
seems fixed in #1281
